context/app/static/js/components/detailPage/BulkDataTransfer/NotLoggedIn/ProtectedData/Links.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+
+import { useFlaskDataContext } from 'js/components/Contexts';
+import OutboundIconLink from 'js/shared-styles/Links/iconLinks/OutboundIconLink';
+import { SecondaryBackgroundTooltip } from 'js/shared-styles/tooltips';
+import { InfoIcon } from 'js/shared-styles/icons';
+import { LinkContainer, StyledLink, LinkTitle } from './style';
+
+function GlobusAccess() {
+  const {
+    entity: { dbgap_study_url, dbgap_sra_experiment_url },
+  } = useFlaskDataContext();
+
+  const LinksContentArray = [
```

Comment:
```suggestion
  const linksContentArray = [
```

Capital-case (aka PascalCase) is reserved for components.